sdk/storage/Azure.Storage.DataMovement/src/LocalFileStorageResource.cs

```diff
@@ -94,13 +94,18 @@ protected internal override Task<StorageResourceReadStreamResult> ReadStreamAsyn
         /// Creates the local file.
         /// </summary>
         /// <param name="overwrite"></param>
+        /// <param name="cancellationToken"></param>
         /// <returns></returns>
-        internal Task CreateAsync(bool overwrite)
+        internal Task CreateAsync(bool overwrite,
```

Comment:
this method looks to actually just not be async, and I don't see that this is an override of a base class method. You can probably just make this method sync?